contract/vm.go

```diff
@@ -722,11 +658,7 @@ func (ce *executor) rollbackToSavepoint() error {
 	}
 
 	var err error
-	for id, v := range ctx.callState {
-		if v.prevState != nil && len(v.prevState.GetCodeHash()) == 0 &&
-			len(v.curState.GetCodeHash()) != 0 {
-			ctx.bs.RemoveCache(id)
-		}
+	for _, v := range ctx.callState {
```

Comment:
Can you explain the above removal?

Reply:
It appears to mean: if there was not a contract on that address and now there is one, revert the deploy

Reply:
it`s my miss. i will recover that.

Reply:
this logic is recovered using isDeploy flag.